src/services/database.js

```diff
@@ -29,6 +30,13 @@ function addSummary(blogpost) {
   return blogpost;
 }
 
+function convertMarkdown(newBlogpost) {
+  const converter = new showdown.Converter();
+  newBlogpost = converter.makeHtml(newBlogpost);
```

Comment:
It is a better practice to never change the function parameters. Instead, you can create a new variable to return.
For this function, you can write an awesome unit test ;)

Reply:
Ok, I changed the function but for the unit test I would need some more input ;)

Reply:
You export this function and import it in a test file. As an input parameter you pass a markdown string and expect a valid html as an output. For example, you pass 
`**Something bold**` and expect `<b>Something bold</b>`